test/system/125-import.bats

```diff
@@ -47,6 +47,13 @@ load helpers
 # Integration tag to catch future breakage in tar, e.g. #19407
 # bats test_tags=distro-integration
 @test "podman export, alter tarball, re-import" {
+    # FIXME: tar < 1.35 is broken. The second tar command below
+    # barfs with "Skipping to next header" then "Exiting with failure status"
```

Comment:
DANGER WILL ROBINSON. This really creeps me out; much like #19407 this is almost certainly a bug in `tar`, but the other way around: last time, old-tar worked and new-rawhide-tar was broken; now new-tar works and old-tar is broken, and all I did was change a source image. I can't tell yet if this bug was intentionally fixed in 1.35, or if it's just pure happenstance that 1.35 passes. I will be looking into this today.